burn-core/src/module/param/id.rs

```diff
@@ -1,6 +1,7 @@
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Hash, PartialEq, Eq, Clone, Serialize, Deserialize)]
+
+#[derive(Debug, Hash, PartialEq, Eq, Clone, Serialize, Deserialize, Ord, PartialOrd)]
```

Comment:
Why the `ParamId` needs to implement `Ord` and `PartialOrd`?

Reply:
This is needed for BTreeMap which is used instead of HashMap. HashMap requires randomization and not supported with no_std.